### PR TITLE
Update to grpc 1.2.0 release

### PR DESCRIPTION
**Pull request by stanley-cheung**

### CODE REVIEW

**Comment by stanley-cheung** [comment]
@jtattermusch friendly ping

**Review by jtattermusch — APPROVED** [review]
LGTM.

File: Formula/grpc.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Grpc < Formula
```
Comment (jtattermusch) [inline Formula/grpc.rb:1]: Btw, it looks like that grpc formula is no longer depending on google-protobuf  but on protobuf.  Not sure if that's a good thing or not.

Comment (stanley-cheung) [reply Formula/grpc.rb:1]: That's a good point. Actually I have submitted our grpc formula to the [homebrew-core](https://github.com/Homebrew/homebrew-core/blob/master/Formula/grpc.rb) repository and there it seems to be automatically pushed forward each release (i.e. I didn't do anything and the formula got updated to 1.2.0). There it also depends on `protobuf` formula not `google-protobuf`, and people seem to be having no issue using that formula. In fact it's easier for user as they don't have to tap `grpc` anymore. 

So is there a reason we need to maintain this repository?

Comment (jtattermusch) [reply Formula/grpc.rb:1]: Good point, let's see if we can deprecate this repo.

**Pull request merged**